tests/ops/functions/test_is_hermitian.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the qml.is_hermitian function
+"""
+import pytest
+
+import pennylane as qml
+
+hermitian_ops = (
+    qml.Identity(0),
+    qml.Hadamard(0),
+    qml.PauliX(0),
+    qml.PauliY(0),
+    qml.PauliZ(0),
+    qml.CNOT([0, 1]),
+    qml.CZ([0, 1]),
+    qml.CY([0, 1]),
+    qml.SWAP([0, 1]),
+    qml.CSWAP([0, 1, 2]),
+    qml.Toffoli([0, 1, 2]),
```

Comment:
We should probably update the `is_hermitian` properties for these ops.  Looks like they never got set to `True`.

Reply:
Done!